src/morris_sensitivity.jl

```diff
@@ -1,15 +1,23 @@
-function generate_design_matrix(prob::DEProblem,p_range,p_steps,k = 10)
+struct Matrix
```

Comment:
probably not a good idea to shadow this

Reply:
It probably would be easier to just use a VectorOfArray from RecursiveArrayTools? Then you'll get all of the indexing and conversion features for free.

Reply:
This is actually to store the matrix and the spread, I didn't understand what you meant by shadow here?

Reply:
Shadowing is when you override a name from Base, making it unavailable without prefixing. `Matrix` is defined and exported in Base Julia. This won't warn, but it will error if someone tries to use `Matrix` the normal way.